app/components/developer_profile_action_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="mt-4 flex space-x-3 md:mt-0">
+  <% if user&.developer %>
```

Comment:
Ok, last change (I missed this last time, sorry). Can this be moved to the component and not the view? `add?` or `new?` method on the component could work.